hack/validate-migration.sh

```diff
@@ -314,28 +324,23 @@ main() {
         return 1
     fi
 
-    local -a migrations_files
-    mapfile -t migrations_files < <(list_migration_files)
+    local output
+    output="$(list_migration_files)"
 
-    if [[ ${#migrations_files[@]} -eq 0 ]]; then
+    if [ -z "$output" ]; then
         echo "No migration."
         exit 0
     fi
 
-    local -r n=${#migrations_files[@]}
-    if [[ $n -gt 1 ]]; then
-        echo "error: found $n migration files. Please ensure to include a single migration file per time." >&2
-        exit 1
-    fi
-
     prepare_pipelines
 
-    local -r file_path=${migrations_files[0]}
-    check_pass_shellcheck "$file_path"
-    check_migrations_is_in_task_version_specific_dir "$file_path"
-    check_version_match "$file_path"
-    check_apply_on_pipelines "$file_path"
-    check_apply_in_real_cluster
+    for migration_file in $output; do
+        check_pass_shellcheck "$migration_file"
+        check_migrations_is_in_task_version_specific_dir "$migration_file"
+        check_version_match "$migration_file"
+        check_apply_on_pipelines "$migration_file"
+        check_apply_in_real_cluster
```

Comment:
Do these work correctly when run multiple times? Do we maybe need to clean something in the `WORK_DIR` before running them again?

Reply:
Updated.